deploy/README.md

```diff
@@ -0,0 +1,158 @@
+# Kubernetes deployment
+
+A helm chart for installing OpenVino Model Server on Kubernetes cluster is provided. By default the cluster contains 
+a single instance of the server but the _replicas_ configuration parameter can be set to create a cluster 
+of any size, as described below. This guide assumes you already have a functional Kubernetes cluster and helm 
+installed (see below for instructions on installing helm).
+
+The steps below describe how to setup a model repository, use helm to launch the inference server and then send 
+inference requests to the running server. 
+
+## Installing Helm
+
+Please refer to: https://helm.sh/docs/intro/install for Helm installation.
+
+## Model repository
+
+If you already have a model repository you may use that with this helm chart. If you don't, you can use any model
+from https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin .
+
+OpenVINO Model Server needs models that it will make available for inferencing. For example you can 
+use Google Cloud Storage bucket:
+```shell script
+gsutil mb gs://model-repository
+```
+
+You can download the model from the link provided above nad upload it to GCS:
+```shell script
+gsutil cp -r 1 gs://model-repository/1
+```
+
+## Bucket permissions
```

Comment:
This section could be simplified if both the cluster and bucket is in the same GCP account. In that case credentials are not required. Just storage API on the cluster level.

Reply:
Done. Added _If the bucket is public or you are going to run Model Server **on the same GCP or AWS account as storage** then no additional changes are needed and you can proceed to _Deploy the Model Server_ section._